### ISSUE TITLE
Multiple labels for multiple lines in display math environment possible?

### ISSUE THREAD

**Issue by gandalfsaxe** [lierdakil/pandoc-crossref#184]
Since you have support for [subfigure labels](https://github.com/lierdakil/pandoc-crossref/blob/master/docs/index.md#subfigures), have you thought about doing the same somehow for multiple lines of display math equations? E.g. for each of the lines:

```markdown
\begin{align}
(a+b)^2 &= (a + b)(a + b) \\
&= a^2 + ab + ba + b^2 \\
&= a^2 + 2ab + b^2
\end{align}
```

Not sure would be best syntax would be. Perhaps a number of labels less or equal to the number if numbered equations above, delimited by line break? So e.g.:

```markdown
\begin{align}
(a+b)^2 &= (a + b)(a + b) \\
&= a^2 + ab + ba + b^2 \\
&= a^2 + 2ab + b^2
\end{align}
{#eq:label1}
{#eq:label2}
{#eq:label3}
```

You could optionally also introduce a dummy label if one wanted to e.g. label equation 1 and 3 but skip 2 in the example above.

**Comment by lierdakil** [comment]
Yeah... that's a very tricky thing to handle correctly. I don't think there's a good way to handle this apart from falling back to LaTeX `\label` syntax, and I *really* don't want to parse LaTeX to be perfectly honest.

**Comment by gandalfsaxe** [comment]
Related to #131, difference being I suggest a "native" pandoc syntax, not parsing LaTeX syntax.

**Comment by peterzjx** [comment]
I have the same need to number and reference multiple labels. My intended target is html5+mathjax only, so I ended up using a custom version of `pandoc-eqnos` here https://github.com/peterzjx/pandoc-eqnos (because, well, I only know python). The syntax is indeed very tricky and I do not have a better way to represent the position of the sublabel. Further discussion is welcomed.

**Comment by ricopicone** [comment]
Just here to say I also would love this feature.
